examples/solids/index.rst

```diff
@@ -338,16 +338,35 @@ Integrating by parts, we arrive at the weak form:
 where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary
 condition written in terms of the reference configuration.
 
-For simplicity, we express here the constitutive law in indicial notation:
+Equation :math:numref:`1st2nd` devises a method to model a hyperelastic solid with a Neo-Hookean constitutive law. Evaluatinig the derivative of this model, yields a forth order tensor:
+
+.. math::
+   :label: mtfs
+
+   \dfrac{\partial \bm P}{\partial \bm F} = \bm S + \left[ \lambda \bm F^{-1} \bm F^{-1} \left(\lambda \log(J) - \mu \right) \left(\bm F^{-1}\bm F^{-1} + \bm C^{-1} \right) \right]
+
+with
+
+.. math::
+   
+   \bm C^{-1} = \bm F^{-1} \bm F^{-T}
```

Comment:
We've already defined `C` and I'm not sure you ever need `F^{-1}` on its own, just `C^{-1}`

Reply:
@jedbrown I have pushed the latest for the symbolic version of the material derivative for hyper FS (equation 35). According to Rich the two tensor product symbols in that equation should be different. Any preference on that?

Reply:
Please have a read through how I would write it
https://libceed.readthedocs.io/en/jeremy-solids-example/examples/solids/index.html#newton-linearization

Please also read my discussion of implementation. I'll add some review notes in `hyperFS.h` because I think this can be made much faster.

Reply:
@jedbrown @jeremylt 
BTW, the derivation of the material derivative in indicial notation is as follows:
![fs-dirv-1](https://user-images.githubusercontent.com/10523693/77261771-9e79ef80-6c56-11ea-88a4-9ea548ab279e.png)
![fs-dirv-2](https://user-images.githubusercontent.com/10523693/77261776-a33ea380-6c56-11ea-8bd8-8d230b5fb6ed.png)
![fs-dirv-3](https://user-images.githubusercontent.com/10523693/77261779-a5a0fd80-6c56-11ea-9a4f-44ac04bd8488.png)

Reply:
I added a derivation of this form in the bottom note here, along with explanation of why the `dS` form (41) is preferable in computation.
![Screenshot_20200323_152545](https://user-images.githubusercontent.com/3303/77364779-91263900-6d1a-11ea-82f0-5df8d4e2c386.png)